java-strings/src/test/java/com/baeldung/stringtolist/ConvertStringToListUnitTest.java

```diff
@@ -0,0 +1,94 @@
+package com.baeldung.stringtolist;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+

```

Comment:
Unnecessary spacing, our Eclipse formatter will clean this up

Reply:
Applied formatter.